pyproject.toml

```diff
@@ -128,3 +136,6 @@ enable = [
 
 [tool.pylint.reports]
 output-format = "colorized"
+
+[tool.black]
+skip-magic-trailing-comma = true
```

Comment:
We could sort this alphabetically, so `[tool.black]` before `[tool.pylint.main]`?